src/backend/metal/src/device.rs

```diff
@@ -1403,10 +1403,15 @@ impl hal::Device<Backend> for Device {
         &self,
         surface: &mut Surface,
         config: hal::SwapchainConfig,
+        old_swapchain: Option<Swapchain>,
     ) -> (Swapchain, hal::Backbuffer<Backend>) {
         self.build_swapchain(surface, config)
     }
 
+    fn destroy_swapchain(&self, _swapchain: Swapchain) -> () {
```

Comment:
Looking at MoltenVK's [implementation](https://github.com/KhronosGroup/MoltenVK/blob/93c524d2f05d7a7f983354b8e53cd4e6812708c7/MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm#L1006), they seem to just destroy it so followed their suit?